OpenOversight/app/static/css/openoversight.css

```diff
@@ -556,3 +560,6 @@ tr:hover .row-actions {
     display:block;
 }
 
+.setup-content {
```

Comment:
I don't see this used, is it needed?

Reply:
It used on the officer identification page: https://github.com/SeattleDSA/OpenOversight/blob/a08396f62dc326210c6ba92b3e6914e3189ec97e/OpenOversight/app/templates/input_find_officer.html#L46

Reply:
Oh! Wild that it didn't already exist :joy: :sob: